### ISSUE TITLE
How can I get the other dreambooth models mentioned in the paper? 

### ISSUE THREAD

**Issue by kelisiya** [DreamEditBenchTeam/DreamEdit#3]
Are they all obtained through fine-tuning?

**Comment by kelisiya** [comment]
And conda creat faild

**Comment by vinesmsuic** [comment]
> And conda creat faild

@kelisiya Can you show the error log?

**Comment by vinesmsuic** [comment]
> Are they all obtained through fine-tuning?

Yes you can get them through fine-tuning a SD model through dreambooth.

**Comment by DreamEditTeam** [comment]
The failure is due to "-e" in "-e git+https://github.com/luca-medeiros/lang-segment-anything.git", we removed it, now it should work. We also add the manually installing script, please try and ask if you meet any problem.

**Comment by DreamEditTeam** [comment]
You can get all the weights at https://vault.cs.uwaterloo.ca/s/EiNjg9yTAKEFgF2
